### DISCUSSION TITLE
PAT supplied to Powershell Invoke-WebRequest, yet Access Denied error

### DISCUSSION THREAD

**Discussion by hanseliezer** [open-metadata/OpenMetadata#16945] [Q&A] [answered]
Seems like a simple problem yet I can't find anything on the docs about this...

I'm simply trying to send a request to the API on my local Docker installation of OM. I'm successful when doing it using `curl` in CMD:

`curl -H "Authorization: Bearer <my bearer token>" -X GET http://<server>:8585/api/v1/metadata/types/name/table -u <username>:<password>`

Yet I keep getting a 407 response when I do it in Powershell:

`Invoke-WebRequest -UseBasicParsing -Headers @{Authorization="Bearer <my bearer token>"} -Uri http://<server>:8585/api/v1/metadata/types/name/table -Method Get -Credential admin`

I think this suggests there is a particular way I have to write my header so that the OM API recognises it? If so, how should I write it?

Thanks in advance

**Comment by Prajwal214** [comment, marked as answer]
<img width="623" alt="image" src="https://github.com/user-attachments/assets/2c0422e4-4f19-4c52-b32c-f948bcca2e61">

https://openmetadata.slack.com/archives/C02B6955S4S/p1720430949065289
